docs/release/support_matrix_v2_0.md

```diff
@@ -1,73 +1,273 @@
+The following subsections provide details on the Linux distributions
+which DAOS Version 2.0 supports on DAOS servers.
+
+Note that all DAOS servers in a DAOS server cluster (also called _DAOS system_)
+must run the same Linux distribution. DAOS clients that access a DAOS server
+cluster can run the same or different Linux distributions.
+
+### CentOS 7 and Red Hat Enterprise Linux 7
+
+With DAOS Version 2.0, CentOS 7.9 and RHEL 7.9 are supported on DAOS servers
+with 2nd gen Intel Xeon Scalable processors (Cascade Lake).
+
+CentOS 7.9 or RHEL 7.9 are *not* supported on DAOS servers
+with 3rd gen Intel Xeon Scalable processors (Ice Lake)
+or newer Intel Xeon processor generations.
+
+Links to CentOS Linux 7 and RHEL 7 Release Notes:
+* [CentOS 7.9.2009](https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7.2009)
+* [RHEL 7.9](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.9_release_notes/index)
+
+CentOS Linux 7 will reach End Of Life (EOL) on June 30st, 2024.
+Refer to the [RHEL Life Cycle](https://access.redhat.com/support/policy/updates/errata/)
+description on the Red Hat support website for information on RHEL support phases.
+
+### CentOS Linux 8
+
+DAOS Version 2.0 supports the latest CentOS Linux 8 release (CentOS 8.5) on the DAOS servers.
+It has also been validated with CentOS 8.3 and 8.4. The DAOS engineering team will accept
+DAOS tickets for DAOS nodes running CentOS 8.3 or 8.4, but may request that the environment
+is updated to the latest CentOS Linux 8 release if that is deemed necessary to fix the issue.
 
-DAOS clients have no specific hardware dependencies.
+Links to CentOS Linux 8 Release Notes:
+* [CentOS 8.3.2011](https://wiki.centos.org/Manuals/ReleaseNotes/CentOS8.2011)
+* [CentOS 8.4.2105](https://wiki.centos.org/Manuals/ReleaseNotes/CentOS8.2105)
+* [CentOS 8.5.2111](https://wiki.centos.org/Manuals/ReleaseNotes/CentOS8.2111)
 
-Some users have reported successful compilation and basic testing of DAOS
-on ARM64 platforms, but those environments are not currently supported.
+CentOS Linux 8 will reach End Of Life (EOL) on December 31st, 2021.
+The DAOS engineering team intends to support Rocky Linux in a future DAOS release.
+DAOS servers running CentOS 8 can then be migrated to either Rocky Linux or RHEL.
 
-## Operating System Support
+### Red Hat Enterprise Linux 8
 
-The DAOS software stack is built on Linux. It is primarily validated
-on [CentOS Linux](https://www.centos.org/centos-linux/)
-and [openSUSE Leap](https://en.opensuse.org/openSUSE:Roadmap).
-CentOS Stream and openSUSE Tumbleweed are not supported.
-Other Linux distributions may be supported in the future.
+DAOS Version 2.0 supports the latest RHEL 8 release (currently RHEL 8.5).
+Customers with Extended Update Support (EUS) for RHEL 8 will also be supported
+with the latest RHEL 8 EUS release (currently RHEL 8.4) on the DAOS servers.
+RHEL 8.3 was not an EUS release, and is no longer supported by Red Hat or DAOS.
```

Comment:
The text says we are **not**. 
People running RHEL do so because they value the enterprise support by RedHat, and we should not support a RHEL OS release that RedHat does no longer support.

Reply:
> The text says we are not.

Indeed.  I see what the text says.  But what we are doing in practice is different.  We are actually building on 8.3 exactly because we were told we need to support it.  It seems that that is not actually the case though and we should move our build to at least 8.4 then, yes?

Additionally, earlier text in the document says: _The DAOS engineering team will accept DAOS tickets for DAOS nodes running CentOS 8.3 …_, which seems to imply some level of support for something that is not even supported by RH, even with EUS.

Reply:
So far our build practice has been to build on an older release, and validate that it works on the newer release. That does not imply we support it on the older release, and the document clearly states that we do not support RHEL 8.3. Yes if 8.3 is no longer needed, we could move the build to 8.4, but the effort to do that is probably bigger than the potential advantages over the current build practice?

We state that we accept DAOS tickets on CentOS 8.3 (and 8.4), because the expectation is that most DAOS tickets are actually DAOS issues, not CentOS issues. There is no reason to reject such tickets solely on the basis that CentOS is not on its latest release level. For the case that the support team determines that there **is** indeed an OS release level dependency, the support document contains the disclaimer that we reserve the right to request an update to the latest CentOS level.